src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```
@@ -486,6 +489,133 @@ trait BronzeTransforms extends SparkSessionWrapper {
+
+    val tmpClusterSnapshotSuccessPath = s"${config.tempWorkingDir}/${apiEndpointTempDir}/success_" + pipelineSnapTS.asUnixTimeMilli
+    val tmpClusterSnapshotErrorPath = s"${config.tempWorkingDir}/${apiEndpointTempDir}/error_" + pipelineSnapTS.asUnixTimeMilli
+
+    landClusterSnapshot(allClusterIDs, pipelineSnapTS.asUnixTimeMilli, tmpClusterSnapshotSuccessPath,
+      tmpClusterSnapshotErrorPath, config)
+
+    logger.log(Level.INFO, " cluster snapshot landing completed")
+
+    if (Helpers.pathExists(tmpClusterSnapshotErrorPath)) {
+      persistErrors(
+        spark.read.json(tmpClusterSnapshotErrorPath),
+        database,
+        clusterSnapshotErrorsTarget,
+        pipelineSnapTS,
+        config.organizationId
+      )
+      logger.log(Level.INFO, "Persist error completed")
+    }
+
+    if (Helpers.pathExists(tmpClusterSnapshotSuccessPath)) {
+      try {
+        val rawDF = deriveRawApiResponseDF(spark.read.json(tmpClusterSnapshotSuccessPath))
+        if (rawDF.columns.contains("cluster_id")) {
+          val outputDF = SchemaScrubber.scrubSchema(rawDF)
+          val finalDF = outputDF.withColumn("default_tags", SchemaTools.structToMap(outputDF, "default_tags"))
+            .withColumn("custom_tags", SchemaTools.structToMap(outputDF, "custom_tags"))
+            .withColumn("spark_conf", SchemaTools.structToMap(outputDF, "spark_conf"))
+            .withColumn("spark_env_vars", SchemaTools.structToMap(outputDF, "spark_env_vars"))
+            .withColumn(s"aws_attributes", SchemaTools.structToMap(outputDF, s"aws_attributes"))
+            .withColumn(s"azure_attributes", SchemaTools.structToMap(outputDF, s"azure_attributes"))
+            .withColumn(s"gcp_attributes", SchemaTools.structToMap(outputDF, s"gcp_attributes"))
+            .withColumn("organization_id", lit(config.organizationId))
+            .verifyMinimumSchema(clusterSnapMinimumSchema)
+          finalDF
+
+        } else {
+          logger.log(Level.INFO, errorMsg)
```

Here instead of logging we should throw a NoNewDataException